### ISSUE TITLE
[ Discussion ] Deployment using Gunicorn

### ISSUE THREAD

**Issue by ridhof** [lalala223/flask-restful-example#5]
Hi! I was trying to find good flask structures and found your repository. 
I read an articles saying that Flask structures mostly are divided into 2 types: divides based on the modules, or single module.

and I see that your structures is using single module one (by making lot of files in resources folder.
so I tried to combine your structures with other structures that separated each modules.

I just started using Flask in late 2019, but I never used the proper or object-oriented-like structures like you did. It's amazing for me to be able to write such a good structures, it thanks to you!

But sure after done with the structures, I have to jump to the next stage, deployment.
Been following some articles about deployment, and trying it out, and I wasn't able to run my app properly. 

I have some questions: To run the Flask on development environment (locally), do you use syntax below right?
```bash
python3 manager.py debug or python3 manager.py run
```

yes, it's successfully executed on my local too, but it's not giving the same result on my DigitalOcean's server. It has an error, especially on 'from abc import ABC'. so I deleted that line  and the args on StandaloneApplication too. is it something that I shouldn't delete?

some articles keep saying to run on server we should do 'gunicorn app:app' but it's not working since app will run create_app(config) that needs config module as args. in the end of the day, I found out that to run on server using gunicorn, I should use 'gunicorn manager:app' and it's successfully executed and running well when I tried to visit on my local browser.

I mean, even it gives me a success, but I don't understand why I should write ':app' there to run the app. I can accept that we should write 'manager' as the .py file that calling our main Flask app. but 'app' itself. Running 'python3 manager.py run' on my local, it will create a new sqlite db for me when I add 'db.create_all()' inside create_app(). but calling 'gunicorn manager:app' doesn't creating a new sqlite db on my server. should I call the db init migrate and etc on server to create one?

I'm sorry if it is too long, I just don't understand anything at all. Thanks a lot for the examples and your time reading my questions!

References I used:
[Kickstarting Flask on Ubuntu – Setup and Deployment](https://realpython.com/kickstarting-flask-on-ubuntu-setup-and-deployment/)
[Deploy flask app with Nginx using Gunicorn](https://medium.com/faun/deploy-flask-app-with-nginx-using-gunicorn-7fda4f50066a)

**Comment by naihuhu** [comment]
I think it's about the problems caused by the version of the gunicorn package. What is the version number of the gunicorn package you are using?

**Comment by ridhof** [comment]
I'm using latest version of gunicorn package.
I think it was due to my lack of fundamental on Flask or Python. But don't worry, I successfully deployed 2 apps without any necessary problems.

Thanks a lot for the structure example, it helps me understanding it's fundamental a bit.
I've been using your structure on 2 projects, one is here [https://github.com/ridhof/twitter-but-flask](https://github.com/ridhof/twitter-but-flask), and the second one is on private repository.
